webapp/src/pages/ResponsePage/ResponsePage.js

```diff
@@ -482,41 +481,55 @@ class ResponsePage extends Component {
         };
     };
 
+    renderActionEditor() {
+        const action_editor = this.state.applicationData.reviewers.find(reviewer => reviewer.is_action_editor === true)
+        if (action_editor) {
+            return <div className="reviewer">
+                        <label>Action Editor</label>
+                        <div>
+                        <p>{action_editor.user_title} {action_editor.firstname} {action_editor.lastname}</p>
+                            
+                        </div>
+                    </div>
+        }
+    }
 
     // Reviews
     // Render Reviews
     renderReviews() {
 
-        if (this.state.applicationData) {
-            if (this.state.applicationData.reviewers) {
-                const reviews = this.state.applicationData.reviewers.map((val, index) => {
-                    let num = index + 1;
-                    //   {"reviewer_user_id": 4, "user_title": "Mr", "firstname": "Joe", "lastname": "Soap", "status": "completed"},
+        if (this.state.applicationData.reviewers) {
```

Comment:
This should be `if (this.state.applicationData && this.state.applicationData.reviewers)` in case applicationData is null (which is what the code was previously trying to do in a very ugly way). In fact, while you're editing this code, it would be better to structure it in the form of a pre-condition to reduce the nesting, so you can do:

``` 
if (! this.state.applicationData || ! this.state.applicationData.reviewers) {
  return <div/>
}

[...]  //Rest of your code now less nested
```